src/state_machine/coordinator/fire.rs

```diff
@@ -209,179 +209,6 @@ impl<Aggregator: AggregatorTrait> Coordinator<Aggregator> {
         }
         Ok((None, None))
     }
-    /// Process the message inside the passed packet
-    pub fn process_message(
-        &mut self,
-        packet: &Packet,
```

Comment:
We should keep this around and use it in the `process` function.

Reply:
Done